make.config

```diff
@@ -13,7 +13,7 @@ INSTALLDIR := $(ROOT)/_install
 ################################################################################
 
 # Whether DFTB+ should support MPI-parallelism
-WITH_MPI := 0
+WITH_MPI := 1 
```

Comment:
Please revert to 0

Reply:
Yes. The make.config slipped into the commit. I have to revert to original. I think it should go into a modified gitignore file.